### ISSUE TITLE
Change License

### ISSUE THREAD

**Issue by wooferzfg1** [LiveSplit/LiveSplit#349]
It seems like the MIT License doesn't require open source for modifications, which is probably something that we want. We should look into a different license (possibly GPL).

**Comment by wooferzfg** [comment]
I don't think we can actually do this because of everyone who has already contributed to this project under the existing license. The MIT license is slightly problematic, however, because we use Xlio, which is licensed under GPL.
